test-fixtures/generate-metrics.sh

```diff
@@ -4,8 +4,8 @@ set -e
 
 BUILD_LOG_FILE="$1"
 TYPE_LOG_FILE="$2"
-THREESHOLD_UNIT_TEST=30
-THREESHOLD_XCUITEST=30
+THREESHOLD_UNIT_TEST=20
+THREESHOLD_XCUITEST=19
```

Comment:
Tacking this on because I forgot to update it in the PR to drop iOS 13 and it looks like the number has dropped a good bit in that time too.